framework/src/mesh/MooseMesh.C

```diff
@@ -2140,7 +2163,8 @@ MooseMesh::effectiveSpatialDimension() const
 {
   const Real abs_zero = 1e-12;
 
-  // See if the mesh is completely containd in the z and y planes to calculate effective spatial dim
+  // See if the mesh is completely containd in the z and y planes to calculate effective spatial
+  // dim
```

Comment:
Hehe - I know what happened here. You indented it and saved, then unindented it. This has happened to me many times.